frontend/src/pages/profile/Profile.tsx

```diff
@@ -112,20 +127,61 @@ const Profile = () => {
               openEditPopUp={handleOpenEditPopUp}
               personnel={personnel}
             />
+            <DialogUI
+              open={openEditPopUp}
+              onClose={handleOpenEditPopUp}
+              handleOpen={handleOpenEditPopUp}
+              title={'Edit Notes'}
+            >
             <ProfileEditForm
               personnel={personnel}
               open={openEditPopUp}
               handleOpenEditPopUp={handleOpenEditPopUp}
               updatePersonnel={updatePersonnel}
             />
+            </DialogUI>
             <ProfileFunctions functions={functions} personnel={personnel} />
             <Scheduler
               name={personnel.firstName}
               availability={availability}
               onChangeAvailabilityDates={onChangeAvailabilityQuery}
               openSchedulerDialog={openSchedulerDialog}
             />
-          </div>
+            <ProfileNotes
+              personnel={personnel}
+              handleOpenEditNotes={handleOpenEditNotes}
+              handleOpenEditCoordinatorNotes={handleOpenEditCoordinatorNotes}
+            />
+            <DialogUI
+              open={openEditNotes}
+              onClose={handleOpenEditNotes}
+              handleOpen={handleOpenEditNotes}
+              title={'Edit Notes'}
+            >
+              <EditNotes
```

Comment:
I feel like this could be simplified a bit more, ie. instead of two <EditNotes> to have one that accepts which field is to be edited.

I'm not fussed though. However, can you move the <DialogUI> components outside of this div? In the same part of the tree that the <Dialog>s are in?